grakn-engine/src/main/java/ai/grakn/engine/controller/response/KeyspaceResponse.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.engine.controller.response;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.google.auto.value.AutoValue;
+
+import javax.annotation.CheckReturnValue;
+
+/**
+ * <p>
+ *     Response object representing {@link ai.grakn.Keyspace}
+ * </p>
+ *
+ * @author Filipe Peliz Pinto Teixeira
+ */
+@AutoValue
+public abstract class KeyspaceResponse {
+
+    @CheckReturnValue
+    @JsonValue
+    public abstract String getValue();
+
+    @CheckReturnValue
+    public static KeyspaceResponse of(String value){
+        return new AutoValue_KeyspaceResponse(value);
```

Comment:
This class should strictly be a wrapper around `Keyspace` itself, not around the enclosed `String`.

Reply:
If we keep this class then I will make the change.